### DISCUSSION TITLE
Error with yaml import of models from a1111

### DISCUSSION THREAD

**Discussion by Bary-Onyx** [comfyanonymous/ComfyUI#462] [Q&A] [answered]
I'm trying to use the extra model paths yaml, as the instructions state but am getting an error at line 8, column 16. Which is the first colon just after checkpoints as shown, how do I solve this problem or what have I done wrong. 
It does it for both cpu and nvidia gpu, any help would be much apricated

 
![error snippet](https://user-images.githubusercontent.com/130361714/230888579-f8200b0c-a105-47ba-81df-138142c7969a.PNG)
![yaml snippet](https://user-images.githubusercontent.com/130361714/230887824-94d071b0-ecab-491c-9425-8f423b0c9e0e.PNG)

**Comment by comfyanonymous** [comment, marked as answer]
https://github.com/comfyanonymous/ComfyUI/blob/master/extra_model_paths.yaml.example

That line after a111: needs a base_path:

**Reply by Bary-Onyx** [reply]
I added back the "base_path:" and now it works thank you
